src/cairoUtilFuncGen/memory/memoryWrite.ts

```diff
@@ -49,10 +49,15 @@ export class MemoryWriteGen extends StringIndexedFuncGen {
     const cairoTypeToWrite = CairoType.fromSol(typeToWrite, this.ast);
     if (cairoTypeToWrite instanceof CairoFelt) {
       return this.requireImport(...WM_WRITE_FELT, inputs, outputs);
-    } else if (
-      cairoTypeToWrite.fullStringRepresentation === CairoUint256.fullStringRepresentation
-    ) {
-      return this.requireImport(...WM_WRITE256, inputs, outputs);
+    }
+
+    if (cairoTypeToWrite instanceof CairoUint) {
+      this.requireImport(
```

Comment:
In the previous version there was a `return` statement here. Do we need to add one now? It appears you are creating two different functions

Reply:
Good catch!